pkg/kubectl/cmd/util/editor/doc.go

```diff
@@ -0,0 +1,58 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package editor // import "k8s.io/kubernetes/pkg/kubectl/cmd/util/editor"
+
+// Package editor is used to complete edit options and open an editor
```

Comment:
What are these commented codes for?

Reply:
These commented codes are used as a example for useage and they maybe make a confusion. I will adjust them.